app/views/pages/about.html.erb

```diff
@@ -40,7 +40,7 @@
 
 <div class="messageBody">
   <%= raw t('pages.about.message_text_three', {
-    linebreak_link: link_to(t('pages.about.linebreak'), 'https://linebreak.co', target: 'blank'),
+    linebreak_link: t('pages.about.linebreak'),
```

Comment:
Nice! In addition to this, let's also update the translation key itself. 

We'll want to replace `%{linebreak_link}` in this [file and other equivalent translation files](https://github.com/ifmeorg/ifme/blob/master/config/locales/en.yml#L394) with just Linebreak. Then we can delete this [line](https://github.com/ifmeorg/ifme/blob/master/config/locales/en.yml#L395).

Reply:
I have updated the .yml files accordingly :)